File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/androidx/NetworkResponse.java
package com.jdroid.android.sample.androidx;

public class NetworkResponse {

	private String id;
	private String value;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getValue() {
		return value;
	}

	public void setValue(String value) {
		this.value = value;
	}

	@Override
	public String toString() {
		final StringBuffer sb = new StringBuffer("NetworkResponse{");
		sb.append("id='").append(id).append('\'');
		sb.append(", value='").append(value).append('\'');
		sb.append('}');
		return sb.toString();
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/androidx/NetworkResponse.kt
package com.jdroid.android.sample.androidx

class NetworkResponse {

    lateinit var id: String
    var value: String? = null

    override fun toString(): String {
        val sb = StringBuffer("NetworkResponse{")
        sb.append("id='").append(id).append('\'')
        sb.append(", value='").append(value).append('\'')
        sb.append('}')
        return sb.toString()
    }
}
